ComfortZoneApp/app/src/main/java/com/example/comfortzone/notification/NotificationUtil.java

```diff
@@ -97,4 +102,28 @@ public static boolean isNotificationEnabled(Context context) {
         return sharedPref.getBoolean(KEY_IS_NOTIFICATION, true);
     }
 
+    @NonNull
+    public static StringBuilder formatPickedTime(int pickedHour, int pickedMinute) {
```

Comment:
I recommend looking into `SimpleDateFormat` to represent the time: https://developer.android.com/reference/java/text/SimpleDateFormat

just create a Date object out of the picked hour and minute, then you can format the time properly